src/util/gamedatas/gameBuildingsTypes.js

```diff
@@ -126,6 +150,10 @@ export const gameBuildingsTypes = [
       {
         id: '62e0e366663a9f95727518c0',
         resourceType: 'energy',
+        calculation: {
+          value: 20,
+          pow: 1.1,
+        },
       },
     ],
     description:
```

Comment:
I cannot comment on the following file, so I do it here instead:

`src/state/slice/buildings/buildingInBuildSlice.js`

You have several code quality issues in that file, please fix them before merging :)